modules/localization/common/localization_gflags.cc

```diff
@@ -65,6 +65,9 @@ DEFINE_bool(enable_gps_timestamp, false,
 
 // msf parame
 DEFINE_string(local_map_name, "local_map", "The path of localization map.");
+DEFINE_string(local_map_config_file,
```

Comment:
Let's use protobuf text instead of xml. We'd better keep config format consistent. Thanks.